src/site/stages/build/drupal/page.js

```diff
@@ -165,6 +165,35 @@ function generateBreadCrumbs(pathString) {
   return entityUrlObj;
 }
 
+// Generate breadcrumbs from GraphQL breadcrumbs
+function generateDrupalBreadCrumbs(entityUrl) {
+  const entityUrlObj = {
+    breadcrumb: [],
+    path: entityUrl.path,
+  };
+
+  if (typeof entityUrl.breadcrumb !== 'undefined') {
```

Comment:
is it necessary to do this`!== 'undefined'`, or can we just check for falsiness like `if (!entityUrl.breadcrumb)`?

Also, per @acabouet's explanation, maybe there's a more descriptive function name than `generateDrupalBreadCrumbs`. Or maybe it would make sense for this function to live in `vets-website/src/site/stages/build/drupal/health-care-region.js`, since that seems to be where the bulk of the Pittsburgh logic lives?

Reply:
Doing the `typeof` check just isn't something I typically see unless `null` or `0` are other special cases. I think `(!entityUrl.breadcrumb)` is a little cleaner.

Hm, so should we remove lines 178-181 then?

> The generateBreadcrumbs() function does not just apply to Pittsburgh content, but other non-Drupal content, as well. So, it probably doesn't make sense to move it to vets-website/src/site/stages/build/drupal/health-care-region.js.

Does `health-care-region` generate more than just the Pittsburgh content though? What I meant was it might make sense to move `generateDrupalBreadCrumbs ` into `health-care-region`  as a local `generateBreadcrumbs` function, since it seems like this function is unique to Pittsburgh health care as well as the `health-care-region` file. So it might help centralize the logic a tiny bit.

Reply:
could use `if (!('entityUrl' in breadcrumb))` though `undefined` should always evaluate falsey and I think we prefer not to differentiate between `undefined` and `null` because we haven't applied a consistent definition to them. e.g. if resetting a key, use `delete myObject.key` not `myObject.key = undefined` or `myObject.key = null`

Reply:
I don't think we need this condition at all actually, since we do it in the calling [function](https://github.com/department-of-veterans-affairs/vets-website/pull/10298/files#diff-a258c6b34aa79768ea36c7a33eb1a445R175). Or am I reading wrong?
 
These are the changes I'm interested in -

1. Remove the duplicate if-statement if necessary
1. Remove lines 178 - 181. 
    - I don't totally understand why these aren't required but I'm trusting your judgement on that.
2. Create some distinction between `generateDrupalBreadCrumbs` and `generateBreadCrumbs`, such as by name or file location.
    - These are both used during the Drupal build step, so it's unclear to me. Maybe this should be renamed `rewriteBreadcrumb`?